### DISCUSSION TITLE
Embed links take much longer to update vs original report links

### DISCUSSION THREAD

**Discussion by salmad** [datapane/datapane#176] [Help] [answered]
Hi everyone,

I noticed a weird behaviour when I update the report (change the underlying data). The embed links seem to take singficantly longer to update (may take minutes!), while the  link to the main page updates reasonably well. The slowest to update is when I select a block by its name. I don't think it is an expected behaviour. Let me know how I can fix it.

- Fastest to update with respect to new data: https://datapane.com/u/salim11/reports/E7opMJk/crypto-report-from-salim
- Medium: https://datapane.com/u/salim11/reports/E7opMJk/crypto-report-from-salim/embed/
- Slowest to update https://datapane.com/u/salim11/reports/E7opMJk/crypto-report-from-salim/embed/?block_name=table

Thanks
Salim

**Comment by mands** [comment, marked as answer]
Hi @salmad ,

This is due to caching behaviour on embed links - we cache these pages for 15m to help keep the site running reliably when serving many requests. We'll make sure to update the docs to make this clearer in the future.
